sdks/python/apache_beam/io/gcp/big_query_query_to_table_pipeline.py

```diff
@@ -50,6 +50,9 @@ def run_bq_pipeline(argv=None):
                       help='Output BQ table to write results to.')
   parser.add_argument('--kms_key', default=None,
                       help='Use this Cloud KMS key with BigQuery.')
+  parser.add_argument('--gs_location',
+                      default=None,
+                      help='GCS bucket location to use to store files.')
```

Comment:
Is it clear for the users of this module which files we are talking about?

Reply:
Improved documentation.